rotkehlchen/tests/db/test_db_accounting_rules.py

```diff
@@ -209,3 +211,57 @@ def test_missing_accounting_rules_accounting_treatment(database: 'DBHandler') ->
     assert not all(
         db.missing_accounting_rules([swap_event_spend, swap_event_receive, swap_event_fee]),
     )
+
+    # test missing accounting rules for non trade events
```

Comment:
1. Wouldn't it be best toadd a different test? This is a separate logical test.
2. Don't you need also one for something like swap with fee?